docs/development/deployment.md

```diff
@@ -30,7 +30,7 @@ To enable development for a team or individual to test changes on your own clust
 
 There are a set of scripts that help with this, and minimize the changes needed in your forks.
 
-The [development configuration](../../argo-cd-apps/overlays/development) includes a kustomize overlay that can redirect the default components individual repositories to your fork.
+The [development configuration](https://github.com/redhat-appstudio/infra-deployments/tree/main/argo-cd-apps/overlays/development) includes a kustomize overlay that can redirect the default components individual repositories to your fork.
```

Comment:
it's better to have it relative, so it works properly also in forks.

Reply:
The problem is with github.io website which does not show directories afaik. It is done similarly to my way in https://github.com/redhat-appstudio/infra-deployments/blob/07d3ebacf22f17e5f96f4cabac32d95c000ec82c/docs/development/spi.md?plain=1#L7.

Reply:
your example is pointing to different git repository

Reply:
ok, I see it now

Reply:
fyi: there are different examples that point to this repo https://github.com/redhat-appstudio/infra-deployments/blob/07d3ebacf22f17e5f96f4cabac32d95c000ec82c/docs/deployment/extending-the-service.md?plain=1#L60